assets/model_monitoring/components/feature_attribution_drift/feature_attribution_drift_compute_metrics/spec.yaml

```diff
@@ -2,7 +2,7 @@ $schema: http://azureml/sdk-2-0/SparkComponent.json
 type: spark
 
 name: feature_attribution_drift_compute_metrics
-version: 0.3.22
+version: 0.3.23
```

Comment:
seems some spec.yaml files like this one doesn't need to upgrade, I suppose you use upgrade --all so all spec files are upgraded, but that's ok.

Reply:
Yes, I upgrade all